src/main/java/com/example/playlistsync/youtube/PlaylistAPI.java

```diff
@@ -0,0 +1,20 @@
+package com.example.playlistsync.youtube;
+
+import com.example.playlistsync.youtube.model.PageInfo;
+import com.example.playlistsync.youtube.model.PlaylistItem;
+import lombok.Data;
+
+import java.util.ArrayList;
+
+@Data
+public class PlaylistAPI {
+
+    private String kind;
```

Comment:
Use lombok to set all variables to private by default instead of using private keyword each time you add variable